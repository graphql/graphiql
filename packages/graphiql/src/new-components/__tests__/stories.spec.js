import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({});
