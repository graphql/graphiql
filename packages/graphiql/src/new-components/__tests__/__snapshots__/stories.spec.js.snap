// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Layout With Full Screen Panel 1`] = `
<main
  className="css-1ox8a0q-Layout"
>
  <div
    className="css-4ddx2x-Card"
  >
    <nav
      className="css-5tspew-Nav"
    >
      <button
        className="css-1ijr7ps-Item"
      >
        üêΩ
      </button>
      <button
        className="css-1ijr7ps-Item"
      >
        üë®‚Äçüåæ
      </button>
      <button
        className="css-1ijr7ps-Item"
      >
        üêù
      </button>
    </nav>
  </div>
  <div
    className="css-neblj2-Layout"
  >
    <div
      className="css-uekhf3-Card"
    >
      <div
        className="css-e025bq-List"
      >
        <div
          className="css-ypzwgl-ListRow"
        >
          Woooo
        </div>
      </div>
    </div>
  </div>
  <div
    className="css-l9qndh-Layout"
  >
    <div
      className="css-nitczu-Card"
    >
      <div
        className="css-e025bq-List"
      >
        <div
          className="css-ypzwgl-ListRow"
        >
          Input
        </div>
      </div>
    </div>
    <div
      className="css-nitczu-Card"
    >
      <div
        className="css-e025bq-List"
      >
        <div
          className="css-ypzwgl-ListRow"
        >
          Response
        </div>
      </div>
    </div>
    <div
      className="css-1ayby34-Card"
    >
      <div
        className="css-e025bq-List"
      >
        <div
          className="css-ypzwgl-ListRow"
        >
          Console/Inspector
        </div>
      </div>
    </div>
  </div>
</main>
`;

exports[`Storyshots Layout With Many Sidebars 1`] = `
<main
  className="css-1ox8a0q-Layout"
>
  <div
    className="css-4ddx2x-Card"
  >
    <nav
      className="css-5tspew-Nav"
    >
      <button
        className="css-1ijr7ps-Item"
      >
        üêΩ
      </button>
      <button
        className="css-1ijr7ps-Item"
      >
        üë®‚Äçüåæ
      </button>
      <button
        className="css-1ijr7ps-Item"
      >
        üêù
      </button>
    </nav>
  </div>
  <div
    className="css-neblj2-Layout"
  >
    <div
      className="css-om1rsp-Card"
    >
      <div
        className="css-e025bq-List"
      >
        <div
          className="css-ypzwgl-ListRow"
        >
          Sidebar
        </div>
      </div>
    </div>
    <div
      className="css-qlah9u-Card"
    >
      <div
        className="css-e025bq-List"
      >
        <div
          className="css-ypzwgl-ListRow"
        >
          aside
        </div>
      </div>
    </div>
    <div
      className="css-qlah9u-Card"
    >
      <div
        className="css-e025bq-List"
      >
        <div
          className="css-ypzwgl-ListRow"
        >
          Another aside
        </div>
      </div>
    </div>
  </div>
  <div
    className="css-l9qndh-Layout"
  >
    <div
      className="css-nitczu-Card"
    >
      <div
        className="css-e025bq-List"
      >
        <div
          className="css-ypzwgl-ListRow"
        >
          Input
        </div>
      </div>
    </div>
    <div
      className="css-nitczu-Card"
    >
      <div
        className="css-e025bq-List"
      >
        <div
          className="css-ypzwgl-ListRow"
        >
          Response
        </div>
      </div>
    </div>
    <div
      className="css-1ayby34-Card"
    >
      <div
        className="css-e025bq-List"
      >
        <div
          className="css-ypzwgl-ListRow"
        >
          Console/Inspector
        </div>
      </div>
    </div>
  </div>
</main>
`;

exports[`Storyshots Layout With Slots 1`] = `
<main
  className="css-1jafrk2-Layout"
>
  <div
    className="css-4ddx2x-Card"
  >
    <nav
      className="css-5tspew-Nav"
    >
      <button
        className="css-1ijr7ps-Item"
      >
        üêΩ
      </button>
      <button
        className="css-1ijr7ps-Item"
      >
        üë®‚Äçüåæ
      </button>
      <button
        className="css-1ijr7ps-Item"
      >
        üêù
      </button>
    </nav>
  </div>
  <div
    className="css-l9qndh-Layout"
  >
    <div
      className="css-nitczu-Card"
    >
      <div
        className="css-e025bq-List"
      >
        <div
          className="css-ypzwgl-ListRow"
        >
          Input
        </div>
      </div>
    </div>
    <div
      className="css-nitczu-Card"
    >
      <div
        className="css-e025bq-List"
      >
        <div
          className="css-ypzwgl-ListRow"
        >
          Response
        </div>
      </div>
    </div>
    <div
      className="css-1ayby34-Card"
    >
      <div
        className="css-e025bq-List"
      >
        <div
          className="css-ypzwgl-ListRow"
        >
          Console/Inspector
        </div>
      </div>
    </div>
  </div>
</main>
`;

exports[`Storyshots Layout With Strings Only 1`] = `
<main
  className="css-1ox8a0q-Layout"
>
  <div
    className="css-4ddx2x-Card"
  >
    nav
  </div>
  <div
    className="css-neblj2-Layout"
  >
    <div
      className="css-om1rsp-Card"
    >
      sidebar
    </div>
  </div>
  <div
    className="css-l9qndh-Layout"
  >
    <div
      className="css-nitczu-Card"
    >
      input
    </div>
    <div
      className="css-nitczu-Card"
    >
      response
    </div>
    <div
      className="css-1ayby34-Card"
    >
      console
    </div>
  </div>
</main>
`;

exports[`Storyshots Lists With Flex Child 1`] = `
<div
  style={
    Object {
      "display": "grid",
      "height": "100vh",
    }
  }
>
  <div
    className="css-e025bq-List"
  >
    <div
      className="css-ypzwgl-ListRow"
    >
      Lists are a vertical stack of components and form the basis of most modules. This one is very long
    </div>
    <div
      className="css-11ix84p-ListRow"
    >
      You normally want 1 flex area that grows forever like this one
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      the end
    </div>
  </div>
</div>
`;

exports[`Storyshots Lists With Stacked Rows 1`] = `
<div
  style={
    Object {
      "display": "grid",
      "height": "100vh",
    }
  }
>
  <div
    className="css-e025bq-List"
  >
    <div
      className="css-ypzwgl-ListRow"
    >
      Title
    </div>
    <div
      className="css-ypzwgl-ListRow"
    >
      Navigation
    </div>
    <div
      className="css-ypzwgl-ListRow"
    >
      Search
    </div>
    <div
      className="css-ypzwgl-ListRow"
    >
      Filter
    </div>
    <div
      className="css-11ix84p-ListRow"
    >
      Actual content
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      <div>
        scroll
      </div>
      Actual content ends here
    </div>
    <div
      className="css-ypzwgl-ListRow"
    >
      Footer
    </div>
    <div
      className="css-ypzwgl-ListRow"
    >
      Footers footer
    </div>
  </div>
</div>
`;

exports[`Storyshots Tabs Tabbar 1`] = `
<div
  className="css-e025bq-List"
>
  <div
    className="css-1kkieog-ListRow"
  >
    <ul
      className="css-1usdqpg-Tabs"
    >
      <li
        className="css-lgj0h8"
      >
        <button
          className="css-8zi662-Tab"
          onClick={[Function]}
        >
          First
        </button>
      </li>
      <li
        className="css-lgj0h8"
      >
        <button
          className="css-141niuh-Tab"
          onClick={[Function]}
        >
          Second
        </button>
      </li>
      <li
        className="css-lgj0h8"
      >
        <button
          className="css-8zi662-Tab"
          onClick={[Function]}
        >
          Third
        </button>
      </li>
    </ul>
  </div>
  <div
    className="css-1kkieog-ListRow"
  >
    <ul
      className="css-1usdqpg-Tabs"
    >
      <li
        className="css-lgj0h8"
      >
        <button
          className="css-8zi662-Tab"
          onClick={[Function]}
        >
          With
        </button>
      </li>
      <li
        className="css-lgj0h8"
      >
        <button
          className="css-141niuh-Tab"
          onClick={[Function]}
        >
          a
        </button>
      </li>
      <li
        className="css-lgj0h8"
      >
        <button
          className="css-8zi662-Tab"
          onClick={[Function]}
        >
          nested
        </button>
      </li>
      <li
        className="css-lgj0h8"
      >
        <button
          className="css-8zi662-Tab"
          onClick={[Function]}
        >
          Component 
          <small
            style={
              Object {
                "background": "yellow",
                "padding": 3,
              }
            }
          >
            2
          </small>
        </button>
      </li>
    </ul>
  </div>
  <div
    className="css-1kkieog-ListRow"
  >
    <div
      style={
        Object {
          "height": 100,
        }
      }
    >
      <ul
        className="css-1usdqpg-Tabs"
      >
        <li
          className="css-lgj0h8"
        >
          <button
            className="css-8zi662-Tab"
            onClick={[Function]}
          >
            Very tall
          </button>
        </li>
        <li
          className="css-lgj0h8"
        >
          <button
            className="css-141niuh-Tab"
            onClick={[Function]}
          >
            tabs
          </button>
        </li>
      </ul>
    </div>
  </div>
</div>
`;

exports[`Storyshots Type Type 1`] = `
<div
  className="css-e025bq-List"
>
  <div
    className="css-ypzwgl-ListRow"
  >
    <h2
      className="css-82xfaz-SectionHeader"
    >
      Title
    </h2>
  </div>
  <div
    className="css-ypzwgl-ListRow"
  >
    <span
      className="css-f0hvcz-Explainer"
    >
      Small explainer text
    </span>
  </div>
  <div
    className="css-ypzwgl-ListRow"
  >
    Normal text
  </div>
</div>
`;
